contracts/interfaces/IStakingPool.sol

```diff
@@ -69,11 +69,6 @@ interface IStakingPool {
     uint32 bumpedPriceUpdateTime;
   }
 
-  struct RewardBucket {
```

Comment:
Packing the values into `RewardBuckets`, didn''t have a great improvement on gas. I changed it to a basic `uint=>uint` mapping for now.